src/sqlfluff/core/parser/grammar/base.py

```diff
@@ -383,11 +404,11 @@ def _trim_elem(seg):
                 return ((), MatchResult.from_unmatched(segments), None)
 
         # Make some buffers
-        seg_buff = segments
-        pre_seg_buff = ()  # NB: Tuple
+        seg_buff = segments  # pragma: no cover
+        pre_seg_buff = ()  # pragma: no cover
 
         # Loop
-        while True:
+        while True:  # pragma: no cover
```

Comment:
This is a lot of uncovered code. Think it can be removed?

Reply:
So, this is all uncovered now because I removed the use of this function in Delimited(). It's now only used used in bracket_sensitive_lookahead_match, which in turn is only used in Bracketed(). I intend to rewrite Bracketed() next, and remove this function entirely, so it's a judgement call, but - I think there's no point trying to capture it in tests, but I also don't think we should remove it, with the caveat that I want rid of the whole function soon

Reply:
No big deal, I was just curious.